CoinDock-api/database/migrations/2022_05_18_080634_create_settings_table.php

```diff
@@ -30,7 +30,8 @@ public function up()
      */
     public function down()
     {
-        Schema::dropForeign('settings_user_id_foriegn');
-        Schema::dropIfExists('settings');
```

Comment:
Is they any specific reason? to remove Schema::dropIfExists('settings');

below code was handle only foreignkey relations. not table drop.

Reply:
changed